lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1786,6 +1786,13 @@ defmodule Kernel.SpecialForms do
 
       &local_function/1
 
+  When the capture operator is given a module name, a remote/external capture
+  is created. Remote captures are faster than local captures. For more
+  information, refer to the ["Functions" section in the Erlang Reference Manual](https://www.erlang.org/doc/system/eff_guide_functions.html#function-calls).
```

Comment:
I think this is a bit misleading, because imported functions would also be external, it isn't about the `m.f/a` syntax:

```elixir
iex> &to_timeout/1
&Kernel.to_timeout/1
```

Perhaps the point could be focused on `&local_function/1` versus `&__MODULE__.local_function/1`?

Reply:
Thanks for the correction! Will do